copulas/univariate/gaussian_kde.py

```diff
@@ -167,9 +167,23 @@ def _fit(self, X):
         self._params = {
             'dataset': X.tolist()
         }
+        self._model = self._get_model()
 
     def _is_constant(self):
         return len(np.unique(self._params['dataset'])) == 1
 
     def _extract_constant(self):
         return self._params['dataset'][0]
+
+    def _set_params(self, params):
+        """Set the parameters of this univariate.
+        Args:
```

Comment:
nit: should there be an extra empty line here?

Reply:
Good catch!